Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "leaflet"
-version = "0.2.3"
-authors = ["slowtec GmbH <post@slowtec.de>"]
+version = "0.3.0"
+authors = ["slowtec GmbH <post@slowtec.de>", "Lewin Probst <info@emirror.de>"]
```

Comment:
Maybe the original authors of leptos-leaflet should also get some credits here or in the readme.

Reply:
Yes of course! I was not able to find details about the author. I already asked about discord and am waiting for response. As soon as I do have a response I will update it.

Sorry that I just added it to `Cargo.toml` hopefully you are ok with that. If you prefer adding it to the `README` I am happy to change that as well.

Reply:
@dgsantana what do you think?

Reply:
@emirror-de , Hi, I'm ok with having my name and email added in the authors of Cargo.toml if that is ok?
Daniel Santana <danielgsantana@gmail.com>

Reply:
If we can get feature parity with the leptos-leaflet "forked" leaflet-rs I'm more then happy to just use this as a dependency.

Reply:
Thank you both @emirror-de and @dgsantana!